### ISSUE TITLE
corpus timeout with use_value_profile=1

### ISSUE THREAD

**Issue by kcwu** [google/clusterfuzz#204]
I have only one fuzz target, which is libfuzzer-asan. I observed, clusterfuzz will run various tasks, like
 - asan fuzzer with full corpus (using use_value_profile=0)
 - asan fuzzer with full corpus using use_value_profile=1
 - asan fuzzer with subset of corpus

(I only discuss fuzzing with full corpus here)

There are some slow input cases in the corpus. Now the situation is:
 - normal asan fuzzer can do fuzz
 - use_value_profile=1 fuzzer timed out during importing the seed corpus.
It always timed out at the same case (the same filename).

In other words, the computing resource on "fuzz with full corpus using use_value_profile=1" is wasted.
They are always timeout.

I am not sure this is "works as intended" or not. I think it's better to remove timeout cases from corpus for use_value_profile=1 fuzzers.

**Comment by Dor1s** [comment]
Yeah, `use_value_profile=1` adds some overhead, but at the same time it gives great benefits in terms of the signal that fuzzing engine gets from a target. The best way to address it is to fix the timeout / make the fuzz target faster.

**Comment by Dor1s** [comment]
> I am not sure this is "works as intended" or not. I think it's better to remove timeout cases from corpus for use_value_profile=1 fuzzers.

The corpus pruning task should be moving "bad" corpus inputs to quarantine, but it happens once a day. Anyhow, the only real solution is to fix the root cause, not to degrade the fuzzing approach.
